packages/appstore/src/components/wallet-locked/wallet-locked.scss

```diff
@@ -0,0 +1,32 @@
+.wallet-locked {
+    &__state {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        max-width: 79.2rem;
+        margin: 14rem auto 0;
+
+        &__title {
+            margin-bottom: 0.8rem;
+        }
+    }
+
+    .legacy-inline-message {
```

Comment:
@heorhi-deriv I think this would effect all the places where <LegacyInlineMessage> is/will be used 🤔.

Reply:
no @aum-deriv it will not be affected to other places since we hide it inside `.wallet-locked` class